init.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+nightly_pattern='^.+(nightly-[0-9]{4}-[0-9]{2}-[0-9]{2}).*$'
+nightly_version=$(grep -E $nightly_pattern ./.circleci/config.yml | head -n 1)
```

Comment:
I hope the problem that we recently had with one of the nightly versions was just limited to that version and in practice, we can just instruct the users the install the very latest version of the nightly. We should change our own config.yml too accordingly.